src/test/Simulation.t.sol

```diff
@@ -969,7 +971,7 @@ contract SimulationTests is DSTest {
         assertLt(dai.balanceOf(bobAddr), amount * auctionPrice / RAY / 5);
     }
 
-    function testTakeLpDaiEthProfit() public {
+    function testTakeLpDaiEthProfit() private { // most LP tokens are getting offboarded
```

Comment:
Same here - seeing NBSP. Please try the same drill as above.

Reply:
done in https://github.com/makerdao/exchange-callees/pull/31/commits/3728627c8870f8e69befb7f7b7f8aaf572eb4c64